### ISSUE TITLE
[bug] When localization is enabled, the system language is not 'en'   The action at the top of the resource details page disappears

### ISSUE THREAD

**Issue by zhouchang2017** [laravel/nova-issues#466]
## When localization is enabled, the system language is not 'en' The action at the top of the resource details page disappears

```
# vendor/laravel/nova/resources/js/views/Detail.vue

# line 32
<div v-if="panel.name.includes('Details')" class="flex items-center mb-3">
```
There is no use of multiple languages

**Comment by dillingham** [comment]
https://github.com/laravel/nova-issues/issues/447

**Comment by davidhemphill** [comment]
This has been fixed in 4420c8ae20f568868ac711812cca893c6a696440 and will be included in an upcoming release.
